s error reporting back to the way it wa

```diff
@@ -0,0 +1,236 @@
+
```

Comment:
I'm not sure what this file is? @dannymendez90

Reply:
neither was i